httpcore5/src/main/java/org/apache/hc/core5/util/Deadline.java

```diff
@@ -60,12 +60,12 @@ public class Deadline {
     /**
      * The maximum (longest-lived) deadline.
      */
-    public static Deadline MAX_VALUE = new Deadline(INTERNAL_MAX_VALUE);
+    public static final Deadline MAX_VALUE = new Deadline(INTERNAL_MAX_VALUE);
 
     /**
      * The minimum (shortest-lived) deadline.
      */
-    public static Deadline MIN_VALUE = new Deadline(INTERNAL_MIN_VALUE);
+    public static final Deadline MIN_VALUE = new Deadline(INTERNAL_MIN_VALUE);
```

Comment:
@jkmcl This breaks binary compatibility. See build failure in Travis C. `URIBuilder` optimization looks good, but two first commits look useless. Please drop them.

Reply:
Sure, I've reverted those 2 commits. Please consider making `Deadline.MIN_VALUE` and `Deadline.MAX_VALUE` final in the future when the API is revised. Thanks